src/patternfly/components/Dropdown/styles.scss

```diff
@@ -24,6 +24,7 @@
 
   // Menu
   --pf-c-dropdown__menu--BackgroundColor: var(--pf-global--BackgroundColor--light-200);
+  --pf-c-dropdown__menu--BorderTransparent: var(--pf-global--BorderColor--transparent);
```

Comment:
I think this variable name is incorrect. I think the name you have for forms css is what it should be: `--BorderColor--transparent` (or maybe even just `--BorderColor`?) @andresgalante please confirm

But you're missing one for `--BorderWidth`, and then these variables should be included in the css below like you did for the other css files.

Reply:
The syntax is `selector--Property` since , `border-transparent` is not a css property, then the variable should read `--BorderColor`